Sources/Lotsawa/Chart.swift

```diff
@@ -47,17 +47,21 @@ extension Chart {
     /// It is arranged to avoid 64-bit alignment, since this will be combined into an `Entry`.
     /// You'd want to swap the elements for optimal performance on big-endian architectures.
     var storage: (
-      /// 16-bit dot position and the low 16 bits of the origin
-      dotPosition_originLow: UInt32,
-      /// The high 16 bits of the origin, 1 bit for isEarley, 14 bit transition or LHS symbol,
-      /// 1 bit for isCompletion.
-      originHi_isEarley_symbol_isCompletion: UInt32)
+      /// The low 16 bits of the origin and a 16-bit dot position.
+      // bits        | xxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxx |
+      // meaning     | originLow        | dotPosition      |
+      originLow_dotPosition: UInt32,
+      /// 1 bit for `isCompletion`, 14 bit transition or LHS `symbol`, 1 bit for `isEarley`, and the high 16 bits of the origin.
```

Comment:
Pls limit your line length.

Reply:
Sure - what's your editorconfig set to? 100?

Reply:
Probably.